src/main/java/com/amuzil/omegasource/magus/skill/conditionals/InputDataBuilder.java

```diff
@@ -0,0 +1,36 @@
+package com.amuzil.omegasource.magus.skill.conditionals;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Designed as a factory class for taking InputDat and turning it into a linked list
+ * for InputModule use.
+ */
+public class InputDataBuilder {
+
+    public static LinkedList<InputData> toInputs(InputData... data) {
+        LinkedList<InputData> out = new LinkedList<>();
+        Collections.addAll(out, data);
```

Comment:
Why are you doing this like this?

Reply:
It's meant to be converting from an array to a list; I can fix that